enhancements/testing/ephemeral-agent-installer-e2e-tests.md

```diff
@@ -0,0 +1,337 @@
+---
+title: ephemeral-agent-installer-e2e-tests
+authors:
+  - "@lranjbar"
+reviewers:
+  - "@rwsu"
+  - "@zaneb"
+  - "@pawanpinjarkar"
+approvers:
+  - "@dhellmann"
+  - "@celebdor"
+api-approvers: # in case of new or modified APIs or API extensions (CRD, aggregated api servers, webhooks, finalizers)
+  - N/A
+creation-date: 2022-02-10
+last-updated: 2022-03-03
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - https://issues.redhat.com/browse/AGENT-20
+---
+
+# Ephemeral Agent Installer e2e Tests
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Create a new e2e testing framework for the new agent based installer project. 
+Currently, for this project there would be potentially two existing e2e frameworks 
+that could be used, [dev-scripts](https://github.com/openshift-metal3/dev-scripts) and [assisted-test-infra](https://github.com/openshift/assisted-test-infra).
+
+After discussing this proposal with the Metal Platform team we will be improving 
+dev-scripts components that overlap in the e2e flow desired for the ephemeral agent installer. 
+
+## Motivation
+
+The new agent based installer project will need an e2e testing framework for validation.
+We should look at the existing frameworks and understand what re-purposing them will 
+entail for this new project.
+
+This enhancement looks at the existing e2e frameworks [dev-scripts](https://github.com/openshift-metal3/dev-scripts) and [assisted-test-infra](https://github.com/openshift/assisted-test-infra)
+closely since this is the previous work in this space. The motivation is to expand on
+what we have learned from those frameworks try to come up with a solution that we can 
+reuse for new installers without these challenges in the future.
+
+### Goals
+
+* This proposal will focus using this new e2e framework in the new ephemeral agent based 
+installer image project.
+
+### Non-Goals
+
+* Re-factoring dev-scripts and assisted-test-infra for reasons not related to the new 
+ephemeral agent based installer image project.
+* There should be some overlap in the pieces that are refactored for use in the new project.
+However it is still a non-goal for outside installer to use this new framework in their e2e 
+tests for the upcoming release. We will plan to create a point of convergence after the
+ephemeral agent based installer project is out of prototyping and MVP phase.
+
+## Proposal
+
+Create an e2e testing framework for the ephemeral agent based installer image.
```

Comment:
Reading through this document, I'm having trouble understanding what the actual plan is. I think there is a plan, but the details aren't making it onto the page in a way that would allow someone else to help with the work.

You could expand this section to make it clear that the plan is to update dev-scripts so it can be used to create dev/test setups and then to manage any new test suite that is created separately from those scripts. This would be a good place to talk about any specific changes that need to be made, too.

Reply:
Made another pass at updating the document to reflect we are using the dev-scripts repo for this.